honeybee_energy_standards/constructionset.py

```diff
@@ -0,0 +1,70 @@
+"""Classmethod for honeybee-energy program type."""
+from __future__ import division
+
+from .lib.constructions import opaque_construction_by_name, window_construction_by_name
+
+
+@classmethod
+def from_standards_dict(cls, data):
```

Comment:
This is a generic comment about all the code that are mainly extension to honeybee-energy. Do we want them to show up in documentation? Should we add a `_` to the name of methods or the name of modules itself as they are not intended to be accessed directly?

Reply:
I purposefully laid out the classmethods in modules that mirror honeybee-energy because I want them to show up in the docs for this library and for people to realize which honeybee-energy objects have this classmethod available. These classmethods should be knowable to users for any case where they want to initialize an object from a standards gem dictionary and there's no way they'll show up any docs unless we make them public.